compiler/hash-token/src/keyword.rs

```diff
@@ -42,40 +44,34 @@ impl fmt::Display for Keyword {
     }
 }
 
-/// A static map of keywords to their enum variants using a
-/// perfect hashing function to quickly lookup the keyword.
-///
-/// @@Future: look into somehow using `gperf` instead?
-static KEYWORDS: phf::Map<&'static str, Keyword> = phf_map! {
```

Comment:
I wonder if all of this extra stuff is worth it for a 2% gain...

Reply:
Yeah, I was wondering about this, it might not be entirely worth it. I need to do some more direct benchmarking to see how much of a win this is actually.